include/yaclib_std/detail/atomic_fence.hpp

```diff
@@ -1,21 +1,23 @@
 #pragma once
 
 #if YACLIB_FAULT_ATOMIC_FENCE == 2
-#  error "YACLIB_FAULT=FIBER not implemented yet"
 
 #  include <yaclib/fault/inject.hpp>
 
+#  include <atomic>
+
 namespace yaclib_std {
 
-inline void atomic_thread_fence(std::memory_order /*order*/) noexcept {
-  YACLIB_INJECT_FAULT((void)0);
+inline void atomic_thread_fence(std::memory_order order) noexcept {
+  yaclib::InjectFault();
```

Comment:
👍

Reply:
Also I thought about disable all injects for atomic_*_fences
What do you think?